packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2184,6 +2186,8 @@ export interface FileUploadAdapter {
     // (undocumented)
     clearFileUploads: () => void;
     // (undocumented)
+    downloadAuthenticatedAttachment?: (attachmentUrl: string) => Promise<string>;
```

Comment:
Make return value like
Promise<DownloadResult>

Then DoownloadResult: {
blobUrl: strin
}

This will give us more flexibility if we want to add more return data to this function

Reply:
Will update this to return `Promise<SomeDownloadedResult>`
Where `SomeDownloadedResult = { blob : blobUrl}`
Something along these lines